MANIFEST.in

```diff
@@ -0,0 +1 @@
+graft src/rowsire/resources
```

Comment:
typo in `roswire`

Reply:
Also, missing line ending may cause issues on certain platforms

Reply:
Weird. The packaging seemed to work even with the misspelling! Fixed (both) now.